api/charms/client.go

```diff
@@ -38,12 +38,13 @@ func (c *Client) IsMetered(charmURL string) (bool, error) {
 
 // CharmInfo holds information about a charm.
 type CharmInfo struct {
-	Revision int
-	URL      string
-	Config   *charm.Config
-	Meta     *charm.Meta
-	Actions  *charm.Actions
-	Metrics  *charm.Metrics
+	Revision   int
```

Comment:
api/charms/ is light on testing.  not sure what should be added.  the apiserver/facades/client/charms is better.

Reply:
Without doubt :dancer:

Reply:
So this is fixed in the #787b32a, I've re-worked the other tests with in the charm package as well, so that we're correctly mocking.